client/extensions/woocommerce/app/dashboard/manage-no-orders.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import BasicWidget from 'woocommerce/components/basic-widget';
+import { getLink } from 'woocommerce/lib/nav-utils';
+import ReadingWidget from 'woocommerce/components/reading-widget';
+import ShareWidget from 'woocommerce/components/share-widget';
+import WidgetGroup from 'woocommerce/components/widget-group';
+
+class ManageNoOrders extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			slug: PropTypes.string.isRequired,
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/d53efa5922db1e46368e0085ee422ed80acfa7de